### PR TITLE
[WIP] Flask sample update

### PR DESCRIPTION
**Pull request by craiglabenz**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: cloud-sql/postgres/sqlalchemy/requirements-test.txt [hunk 1]
```diff
@@ -1 +1,3 @@
+-r requirements.txt
```
Comment (engelke) [inline cloud-sql/postgres/sqlalchemy/requirements-test.txt:1]: This shouldn't be needed. The test runner should install both requirements.txt and requirements-test.txt.

**Review by engelke — APPROVED** [review]
LGTM

**Comment by tmatsuo** [comment]
@craiglabenz Lint build is failing. Do you have access to the logs?

**Comment by craiglabenz** [comment]
@engelke and @tmatsuo -- thanks for the input! I've been a bit distracted recently and haven't been paying attention to this PR. Sorry to accidentally ignore your comments!

I'm still using it as a bit of a staging area, so it's not ready for serious evaluation yet, either.

**Comment by craiglabenz** [comment]
Closing in favor of a different PR.

**Pull request closed without merging**